src/components/common/TmSessionExisting.vue

```diff
@@ -7,7 +7,7 @@
       <h2 class="session-title">
         Use an existing address
       </h2>
-      <a @click="$store.commit(`toggleSessionModal`, false)">
+      <a @click="$emit('close')">
```

Comment:
the we are emitting events is inconsistent :|

Reply:
you mean inline vs method?